### PR TITLE
feat: support realtime toggle update via socket.io

### PR DESCRIPTION
**Pull request by hezean**
**realtime toggle update is not supported in python3.5**, see https://github.com/miguelgrinberg/python-socketio/issues/589

### CODE REVIEW

**Comment by SSebo** [comment]
Good Job! You are so productive! BTW suggest to separate quotation mark replacement to another PR, to make this PR more focus on socketio part.

**Comment by FengShang** [comment]
related to #19

File: featureprobe/internal/deprecated.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    if callable(anything):
+        return "function<{}>".format(anything.__name__)
+    elif isinstance(anything, property):
+        return "property<{}>".format(anything.fget.__name__)
+    # TODO: add more type support
+    else:
+        return None
+
+
+def deprecated(*, since: str = None, successor=None):
```
Comment (hezean) [inline featureprobe/internal/deprecated.py:28]: not used in this pr, but it may be used someday, so keep it?

**Comment by hezean** [comment]
@dengfeige can we start reviewing this pr? it's functionally equivalent with https://github.com/FeatureProbe/server-sdk-node/pull/6

**Comment by hezean** [comment]
ci crashed due to py3.5 not found, please help re-trigger it later, thanks

**Comment by gangb-tech** [comment]
This PR has already been merged into #24.

**Pull request closed without merging**